src/operations/resolvers.tsx

```diff
@@ -0,0 +1,135 @@
+import { Resolvers } from '@apollo/client'
+import { v4 } from 'uuid'
+
+import { GET_COLLECTIONS } from './queries/getCollections'
+import type { Bookmark, Collection } from 'types'
+
+export const localResolvers: Resolvers | Resolvers[] = {
+  Query: {
+    collections: (_root, { id }, { cache }) => {
```

Comment:
@suzubara this resolver has my brain in a STATE and i'd love to pair a bit to talk through this. 

- it's not always triggered  when i would expect it to
- it seems to have no effect on the rendering of my MySpace component in another branch
- it _is_ required to query as expected in GraphiQL

Reply:
Discussed with @suzubara -- leaving in for now and revisiting later